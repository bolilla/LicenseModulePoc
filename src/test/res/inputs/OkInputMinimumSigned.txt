parameter:value
===================== SIGNATURE BEGIN =====================
MCwCFHZRdQTfR2ssAoQC8+HJxHC1Qpn8AhQGbrXwppkkxIlg+libToQj0WzkTA==
=====================  SIGNATURE END  =====================
