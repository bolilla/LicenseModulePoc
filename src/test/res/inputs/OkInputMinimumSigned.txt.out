parameter:value
===================== SIGNATURE BEGIN =====================
MCwCFBWiefGIqOGzkzlC7FEZkkEYaQi8AhROgt/7zMTodCuPLnROI8rMFuGtfg==
=====================  SIGNATURE END  =====================
